src/main/java/pages/AccountCreationPage.java

```diff
@@ -0,0 +1,142 @@
+package pages;
+
+import cucumber.api.DataTable;
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+import java.util.Map;
+
+@Getter
+public class AccountCreationPage {
+    WebDriver driver;
+
+    public AccountCreationPage(WebDriver driver) {
+        this.driver = driver;
+
+        PageFactory.initElements(driver, this);
+        dropDownDay = new Select(dayOfBirth);
+        dropDownMonth = new Select(monthOfBirth);
+        dropDownYear = new Select(yearOfBirth);
+        stateName = new Select(state);
```

Comment:
why do we need all this elements in constructor?

Reply:
I initialize these variables in the constructor at the same time  with all web elements of the page in order to be able to use the method for selecting values from the drop-down, otherwise, I get NullPointerException